src/common/ErrorPage/index.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import ErrorIcon from "../../assets/images/error.svg";
+import { ErrorContainer, StyledImage, ErrorTitle, ErrorBody, ErrorButton } from "./styled";
+
+export default () => (
+    <ErrorContainer>
+        <StyledImage src={ErrorIcon}></StyledImage>
+        <ErrorTitle>Ooops! Something went wrong...</ErrorTitle>
+        <ErrorBody>
+            Please check your network connection<br />
+                and try again
```

Comment:
Tutaj musimy się zastanowić czy ten tekst nie będzie w komponencie głównym Error jako props np. errorInfo, bo mogą spływać różne komunikaty błędów, w zależności od błędu i sytuacji ;) i wtedy też ten <br /> jest raczej średnim pomysłem ;)

Reply:
Pytanie czy chcemy użytkownikowi pokazywać jakie dokładnie mamy błędy? To chyba bardziej info dla developerów a użytkowników obchodzi tylko to że coś nie działa.
Oczywiście można tak zrobić, ale nie wiem czy to nie będzie trochę na siłę.

Reply:
Tak jak Kasia pisze, ten tekst który jest w komponencie to jest jasny i główny komunikat dla użytkownika. Zakładamy, że użytkownik jet nie techniczny i nic go nie obchodzi dlaczego jest błąd, jedyne co ma dostać to instrukcję co ma zrobić w takiej sytuacji a tą jest kliknięcie w przycisk.

Saga wywali błąd w konsoli dla bardziej wtajemniczonych.